pyproject.toml

```diff
@@ -13,6 +13,7 @@ defusedxml = "^0.7.1"
 demisto_sdk = "1.16.0"
 requests = "^2.22.0"
 pre-commit = "^2.20.0"
+flake8 = "^4.0.1"
```

Comment:
why?

Reply:
@ilaner do we **need** it?

Reply:
it will work on both cases, as `flake8` is installed as a part of demisto-sdk.
I think we can add this as dependency to the content, as we use it in content as well (if we someday plan to remove `flake8` from demisto-sdk, it still should remain on content.

Reply:
As none of the tasks here explicitly call flake8, please remove it.